modules/preferences/main/index.coffee

```diff
@@ -286,8 +286,7 @@ class Preferences extends hx.EventEmitter
 
   applyTimezoneOffset: (date, offset) ->
     offset ?= @_.timezoneOffsetLookup(@timezone(), date.getTime()) || 0
-    utc = date.getTime() + (date.getTimezoneOffset() * 60000)
-    new Date(utc + offset * 60 * 60 * 1000)
+    new Date(date.getTime() + -1 * offset * 60 * 60 * 1000)
```

Comment:
This has reversed the desired behaviour of the localization:
```js
const date = new Date(1485333362249)
hx.dateTimeLocalizer().time(date)
// Returns -> "8:36"
hx.preferences.timezone("UTC-08:00")
hx.dateTimeLocalizer().time(date)
// Returns -> "16:36"
```

The desired behaviour is to display the current time relative to the timezone so should return `0:36` in this case.

The localization currently mimics that provided by `moment.js`, where we took a lot of the logic from. We also use moment.js under the hood (if it is available in the browser).

Equivalent Moment.js:
```js
const momentDate = moment(1485333362249)
momentDate.format('H:mm')
// Returns -> "8:36"
momentDate.tz('America/Los_Angeles').format('H:mm')
// Returns -> "0:36"
```

As far as I can see, you're correct in removing where we  were using `date.getTimezoneOffset`, I think removing the `-1 *` should resolve the issue where the behaviour is flipped.

Could you also add a few positive values to the `timezoneOffsets`? 

We should probably also test with different `hx.preferences.timezone` options set as we are using them to determine what the time should be.